src/api/offers.js

```diff
@@ -12,6 +12,8 @@ import ErrorService from '../services/ErrorService'
 
 const develop = process.env.NODE_ENV === 'development'
 
+const CONFIRMATION_PERIOD = 1000 * 60 * 60 * 48
```

Comment:
In the context of eslint these are still magic values:

https://github.com/Integreat/neuburg-backend/pull/28/files#diff-b528c50833f673d233fa7dc20b02600eR18

Reply:
Done